compiler/circle-mpqsolver/CMakeLists.txt

```diff
@@ -1,8 +1,27 @@
 file(GLOB_RECURSE SOURCES "src/*.cpp")
+file(GLOB_RECURSE TESTS "src/*.test.cpp")
+list(REMOVE_ITEM SOURCES ${TESTS})
 
 add_executable(circle-mpqsolver "${SOURCES}")
+target_link_libraries(circle-mpqsolver luci_service)
+target_link_libraries(circle-mpqsolver luci_partition)
```

Comment:
plz check if `luci_partition` is needed in `circle-mpqsolver`

Reply:
it's needed. The code needs `<luci/ConnectNode.h>` without `luci_partition` compilation fails.

Reply:
Now I see that you copied codes from _luci_ for cloning the graph.
Sorry to say this but I cannot accept this;
- I didn't expose the clone_graph as currently it's not fully verified 
- This will make duplicate codes and our SE is counting that score (circle-mpqsolver is not counted yet but it will one day)

If you need a double instance, why not read it twice for two instances?

Reply:
@stamalakhov Could you change the code to import Module, instead of copy in-memory ?

Reply:
Which means to read it from disk any time we want to make a copy?

Reply:
Yes. It would be a bit inefficient, but more stable. Fortunately, some `ModuleCloner::clone` seems unnecessary.

https://github.com/Samsung/ONE/pull/10191/files#diff-a04366e0b2c97ad2017df8d99df403ef63bd4282746b773f9c522bf955f6592bR36

Passing `unique_ptr` (instead of Module *) may reduce unnecessary clones.

Reply:
Ok. I see.